make/Docs.gmk

```diff
@@ -635,6 +635,10 @@ $(foreach m, $(ALL_MODULES), \
   ) \
 )
 
+# Customization for specific files
+
+jdk.javadoc_doc-comment-spec.md_TOC := --toc
```

Comment:
Since this is a method to add any kind of options to pandoc, please call it _OPTIONS instead of _TOC.

Or, if you really want to make it toc-specific, you should do something like:
```
jdk.javadoc_doc-comment-spec.md_TOC := true
```
and check the boolean value for this and add `--toc` as appropriate. (That does seem like a worse solution to me.)